Sources/MapboxMaps/Foundation/BaseMapView.swift

```diff
@@ -93,43 +93,110 @@ open class BaseMapView: UIView, MapClient, MBMMetalViewProvider {
     @IBInspectable var baseURL__: String = ""
     @IBInspectable var accessToken__: String = ""
 
-    /// Returns the camera view managed by this object.
-    public var cameraView: CameraView!
-
     internal var preferredFPS: PreferredFPS = .normal {
         didSet {
             updateDisplayLinkPreferredFramesPerSecond()
         }
     }
 
+    /// Returns the camera view managed by this object.
+    public var cameraView: CameraView!
+
+    /// The map's current camera
+    public var camera: CameraOptions {
```

Comment:
I can't remember if we have a ticket for this or not. But one thing that really gets confusing is that camera is `CameraOptions`. Can one of those be renamed?

In my mine, `CameraOptions` should be a set of properties/options that can manipulate the `Camera`.

Reply:
Let's review naming as part of a separate PR.